spec/controllers/articles_controller_spec.rb

```diff
@@ -1,22 +1,22 @@
 require 'spec_helper'
 
 RSpec.describe ArticlesController, type: :controller do
-  describe '#index' do
+  describe 'GET #index' do
     describe 'show the list to articles saved' do
     end
   end
 
-  describe '#show' do
+  describe 'GET #show' do
     it 'show the articles' do
```

Comment:
`it 'assign the article` this is without `s` because this action is applied only to one object.